packages/ui-bindings/AgentRelationshipRole/AgentRelationshipRoleById.tsx

```diff
@@ -0,0 +1,48 @@
+import { connect } from "react-redux"
+import { gql, graphql, compose } from "react-apollo"
+import { AppState } from "@vflows/store/types"
+import { getActiveLoginToken } from "@vflows/store/selectors/auth"
+
+const query = gql`
+query($token: String, $id: Int){
+  viewer(token: $token) {
+    agentRelationshipRole(id: $id) {
+      id
```

Comment:
Did you consider making a fragment for this to reduce code duplication?

Reply:
Fragment added
:+1: